### PR TITLE
[uss_qualifier] DSS0300 (utm) setup check for basic requirements

### PR DESCRIPTION
**Pull request by Shastick**
First very basic check for DSS0300, inspired from what is done in the NETRID test suites.

### CODE REVIEW

**Review by mickmis — COMMENTED** [review]
LGTM. Some small comments plus the CI that is not passing (missing resource error).

File: monitoring/uss_qualifier/suites/interuss/dss/all_tests.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ name: ASTM DSS tests
 resources:
   f3411v19_dss_instances: resources.astm.f3411.DSSInstancesResource
   f3411v22a_dss_instances: resources.astm.f3411.DSSInstancesResource
+  f3548v21_dss_instances: resources.astm.f3548.v21.DSSInstancesResource
```
[the remark this answers is not in the document]
Comment (Shastick) [reply monitoring/uss_qualifier/suites/interuss/dss/all_tests.yaml:5]: Looks like a leftover from my trial and error to get things running: I'll remove it.

File: monitoring/uss_qualifier/resources/astm/f3548/v21/dss.py [hunk 2]
```diff
@@ -82,6 +91,13 @@ def get_full_op_intent(
             ).operational_intent
         return result, query
 
+    def is_same_as(self, other: DSSInstance) -> bool:
```
[the remark this answers is not in the document]
Comment (Shastick) [reply monitoring/uss_qualifier/resources/astm/f3548/v21/dss.py:94]: It feels a bit weird to override `__eq__` as we're not strictly comparing the instances (ie, `client` might be different in some fundamental way).

On the other hand, the method could be renamed to something clearer such as `has_same_backend` or similar?

**Comment by Shastick** [comment]
Added the action generator. I currently can't get the check to run, a little detail is probably wrong somewhere.

File: monitoring/uss_qualifier/scenarios/astm/utm/dss_interoperability.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+        self.begin_test_step("Test environment requirements")
+        self._test_env_reqs()
+        self.end_test_step()
+
+        self.end_test_case()
+
+        self.end_test_scenario()
+
+        raise ValueError("IT RAN")
```
Comment (Shastick) [inline monitoring/uss_qualifier/scenarios/astm/utm/dss_interoperability.py:64]: Note that this is temporary, I'd like to quickly identify when this test scenario runs or not.

**Comment by Shastick** [comment]
Build passes now, and I confirmed locally that with the current configuration the check is run.

File: monitoring/uss_qualifier/scenarios/astm/utm/dss_interoperability.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+from monitoring.uss_qualifier.common_data_definitions import Severity
+from monitoring.uss_qualifier.resources.astm.f3548.v21.dss import (
+    DSSInstancesResource,
+    DSSInstanceResource,
+    DSSInstance,
+)
+from monitoring.uss_qualifier.scenarios.scenario import TestScenario
+
+VERTICES: List[LatLngPoint] = [
```
Comment (BenjaminPelletier) [inline monitoring/uss_qualifier/scenarios/astm/utm/dss_interoperability.py:16]: In the future, we should accept any inputs like this as resources so the test can be appropriately configured to suit the user (e.g., this wouldn't be a good location to test DSS instances in Europe).  But no need to do everything in a single PR, so no expectation to adjust for this PR.

**Review by BenjaminPelletier — APPROVED** [review]

**Pull request merged**